.infrastructure/helm-chart/todoapp/templates/ingress.yml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Chart.Name }}-ingress
+  namespace: {{ .Values.todoapp.namespace }}
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
+spec:
+  rules:
+  - http:
+      paths:
+      - pathType: Prefix
+        path: /(|$)(.*)
+        backend:
+          service:
+            name: {{ .Chart.Name }}-service
+            port:
+              number: 80
```

Comment:
Fix EoF

Reply:
fixed in [fix EoF, add serviceAccount YAML file, use pv storage value from value.yml](https://github.com/mate-academy/devops_todolist_kubernetes_task_12_helm_charts/pull/30/commits/72ec2793e112758cfac44d2779c6d423be281e55) commit